tests/Aspire.Hosting.Tests/DistributedApplicationTests.cs

```diff
@@ -336,7 +336,9 @@ public async Task VerifyDockerWithEntrypointWorks()
         using var cts = new CancellationTokenSource(Debugger.IsAttached ? Timeout.InfiniteTimeSpan : TimeSpan.FromSeconds(10));
         var token = cts.Token;
 
-        var redisContainer = await KubernetesHelper.GetResourceByNameAsync<Container>(s, "redis-cli", r => r.Status?.EffectiveEnv is not null, token);
+        var redisContainer = await KubernetesHelper.GetResourceByNameAsync<Container>(s, "redis-cli",
```

Comment:
This change is not related to the problem with `AllocatedEndpointAnnotation`, but rather this test was consistently failing on my dev box, so I fixed it.